lib/process-services/process-list/components/process-filters.component.ts

```diff
@@ -58,7 +58,7 @@ export class ProcessFiltersComponent implements OnInit, OnChanges {
 
     /** Toggle to show or hide the filter's icon. */
     @Input()
-    showIcon: boolean = true;
+    hasIcon: boolean;
```

Comment:
this is a BC, we cannot change the name of the input property

Reply:
Why you are changing the process files if the issue is realted to the task ?

Reply:
There is a gap between process-filters html and ts files. We have "showIcon" on ts file and "hasIcon" on html file. I fixed that in process-filters PR.